### DISCUSSION TITLE
"Terms of use" policy?

### DISCUSSION THREAD

**Discussion by gwerbin** [monkeytypegame/monkeytype#1683] [Questions] [answered]
Is there a "terms of use" or "terms of service" policy for Monkeytype? I don't see one on the website. Maybe having such a policy is a good idea, as an extra layer of protection against things like botting, hacking, liability, etc. (specifics will depend on legal jurisdiction).

**Comment by Miodec** [comment, marked as answer]
Yeah, i should probably add something
